### ISSUE TITLE
Indexing and Searching in network locations only shows first result

### ISSUE THREAD

**Issue by NihalShavdia** [adirh3/Fluent-Search#1008]
When searching on network locations, only the first result shows up.

I'm trying to set up a file search tag, "vic" that referencing a network in a different location.
![image](https://github.com/adirh3/Fluent-Search/assets/53282984/52eaae1a-2f6f-4ae2-9543-c80f606ca2f7)

However, when using that tag, I only get results from the first folder in that directory (CAD15) , and then the first folder in there
![image](https://github.com/adirh3/Fluent-Search/assets/53282984/0aa0ce17-bf7d-4146-a7db-3a1b347023a0)
![image](https://github.com/adirh3/Fluent-Search/assets/53282984/ba88a978-670b-49c2-bc5b-54650414af31)
Basically seems like the search is only happening in the first 2 consecutive folders
E.g. if I try to search CAD 16:
![image](https://github.com/adirh3/Fluent-Search/assets/53282984/f70152dd-c58e-4a06-8629-3e25af2576a6)
![image](https://github.com/adirh3/Fluent-Search/assets/53282984/e7c3651e-fbb7-4661-b774-745c1e46da05)

 - Windows 11
 - Fluent Search Version: 1.0.0.42
 - Installation source: Installer 

Same issue when using file search tags, and generic search tags.

**Comment by adirh3** [comment]
This issue has been potentially fixed in version 1.1.0.8. Changelog - https://github.com/adirh3/Fluent-Search/releases/tag/1.1.0.8

**Comment by adirh3** [comment]
Closing as the fix has been released and no further issues were reported. If you're still experiencing this on the latest version, please reopen.
